<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Psychological_Counselor</class>
 <widget class="QDialog" name="Psychological_Counselor">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1579</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Psychological Counselor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLabel" name="picture_1">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>50</y>
     <width>561</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>data/psychological counselor_5.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="emotion">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Emotion Cognition Webcam</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="video_widget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>631</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_input">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>740</y>
     <width>801</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>굴림</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="chat_history">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>350</y>
     <width>801</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>굴림</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>750</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Send message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>720</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;휴먼둥근헤드라인&quot;;</string>
   </property>
   <property name="text">
    <string>Transfer my emotion</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
